src/components/access-keys/AccessKeysDeauthorizeConfirm.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Translate } from 'react-localize-redux'
+
+import FormButton from '../common/FormButton'
+
+import { Input, Form } from 'semantic-ui-react'
+
+const AccessKeysDeauthorizeConfirm = ({ handleConfirmSubmit, handleChange, accountId, confirmStatus, handleConfirmClear, buttonLoader }) => (
+    <Form onSubmit={(e) => handleConfirmSubmit(e)}>
+        <Translate>
+            {({ translate }) => (
+                <Input 
+                    name='accountId'
+                    value={accountId}
+                    onChange={handleChange}
+                    className={confirmStatus ? (confirmStatus === 'success' ? 'success' : 'problem') : ''}
+                    placeholder={translate('login.confirm.username')}
+                    maxLength='32'
+                    required
+                    autoComplete='off'
+                    autoCorrect='off'
+                    autoCapitalize='off'
+                    spellCheck='false'
+                    tabIndex='1'
+                    autoFocus={true}
+                />
+            )}
+        </Translate>
+        <div className='alert-info'>
+            {confirmStatus === 'problem' && `Account name doesn't match`}
```

Comment:
Minor: Seems string is adequate since there's no variable

Reply:
@marcinbodnar shouldn't this use localized string too?

Reply:
@vgrichina Good point. Done in #514